erts/emulator/sys/unix/erl_child_setup.c

```diff
@@ -591,10 +590,17 @@ main(int argc, char *argv[])
                 errno = 0;
 
                 os_pid = fork();
-                if (os_pid == 0)
+                if (os_pid == 0) {
                     start_new_child(pipes);
+                }
 
-                add_os_pid_to_port_id_mapping(proto.u.start.port_id, os_pid);
+                {
```

Comment:
Scoping block is to inline the hash side-effects, but making it clear that the `es` variable won't be used again.